test/package.json

```diff
@@ -58,6 +58,9 @@
     "@payloadcms/ui": "workspace:*",
     "@sentry/react": "^7.77.0",
     "@types/react": "npm:types-react@19.0.0-rc.0",
+    "next": "15.0.0-canary.104",
+    "react": "19.0.0-rc-06d0b89e-20240801",
+    "react-dom": "19.0.0-rc-06d0b89e-20240801",
     "@types/react-dom": "npm:types-react-dom@19.0.0-rc.0",
     "babel-plugin-react-compiler": "0.0.0-experimental-592953e-20240517",
```

Comment:
I'd recommend using the same version of the compiler across all your packages for consistency

Reply:
good catch!